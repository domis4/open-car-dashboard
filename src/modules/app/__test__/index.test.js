import React from 'react'
import { shallow } from 'enzyme'
import App from '../index'

describe('app module index.js', () => {
  it('expects component to match snapshot', () => {
    const wrapper = shallow(<App />).dive()

    expect(wrapper).toMatchSnapshot()
  })
})
