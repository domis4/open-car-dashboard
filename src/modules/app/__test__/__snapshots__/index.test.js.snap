// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`app module index.js expects component to match snapshot 1`] = `
<WithStyles(App)
  classes={
    Object {
      "app": "WithStyles-App--app-02",
      "content": "WithStyles-App--content-03",
      "root": "WithStyles-App--root-01",
      "router": "WithStyles-App--router-04",
    }
  }
  isBootIconVisible={true}
  setBootIconVisible={[Function]}
/>
`;
